provision/acc_provision/templates/aci-operators-crd.yaml

```diff
@@ -10,4 +10,22 @@ spec:
     plural: acicontainersoperators
     singular: acicontainersoperator
   scope: Namespaced
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: CRD to own the lifecycle of ACI objects in the cluster
```

Comment:
I believe as per other CRD examples, the start should not be generic ( like CRD).  Rather like below:
"acicontainersoperator owns the lifecycle of ACI objects in the cluster"

Reply:
likewise for subsequent changes

Reply:
Added in the latest push